package.json

```diff
@@ -36,7 +36,7 @@
     "localforage-observable": "2.0.1",
     "mark.js": "8.11.1",
     "marked": "github:mattermost/marked#87769262aa02e1784570f61f4f962050e07cc335",
-    "mattermost-redux": "github:mattermost/mattermost-redux#47b470894fc7c0cac8181a04b348295b4bf8855b",
+    "mattermost-redux": "git://github.com/mattermost/mattermost-redux.git#MM-27535",
```

Comment:
This should be using the most recent commit hash that includes your redux changes.